Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -226,30 +126,6 @@ public String getServletPathWithHostAsUrl() {
         return list;
     }
 
-    /**
-     * Method returns a valid css file, which is the suggestion unless suggestion is
-     * not available if not available default.css is returned.
-     *
-     * @param cssFileName
-     *            suggested css file
-     *
-     * @return valid css file
-     */
-    public String getCssLinkIfExists(String cssFileName) {
-        FilenameFilter filter = new FileNameEndsWithFilter(".css");
-        List<URI> uris = serviceManager.getFileService().getSubUris(filter, URI.create(CSS_BASE_PATH + CSS_PATH));
-        for (URI uri : uris) {
-            if (uri.toString().equals(cssFileName)) {
-                return cssFileName;
-            }
-        }
-        return CSS_PATH + "/default.css";
-    }
-
-    public String getLogoUrl() {
-        return getServletPathWithHostAsUrl() + "/pages/images/template/kitodo-homepage-logo.svg";
```

Comment:
Please investigate if this file `kitodo-homepage-logo.svg` still used in new design.